scripts/g.extension/g.extension.py

```diff
@@ -155,7 +155,7 @@
 import json
 import xml.etree.ElementTree as etree
 
-if sys.version_info.major == 3 and sys.version_info.minor < 8:
+if sys.version_info.major == 3 and sys.version_info.minor < 8:  # noqa: YTT204
```

Comment:
Wouldn't this be better fixed with the full version comparison from the [doc](https://docs.astral.sh/ruff/rules/sys-version-info-minor-cmp-int/). It's fine if Python 2 falls into this here.

Reply:
```suggestion
if sys.version_info.major == 3 and sys.version_info < (3, 8):
```

I assume you meant that it is fine if Python 2 fails.

Reply:
Isn`t this sufficient?

```suggestion
if sys.version_info < (3, 8):
```

I understood @wenzeslaus comment that we do not have to worry about the fact that Python 2 evaluates to `True` here, because it will fail earlier in many other places anyway.... @wenzeslaus please jump in if I did not get you right...

Reply:
@wenzeslaus we're waiting for a choice between one of the two options before merging.